### ISSUE TITLE
Flutter Web: Text <input> form elements that collect information about the user require the autocomplete attribute (Accessibility)

### ISSUE THREAD

**Issue by jcollins-disney** [flutter/flutter#162136]
### Use case

Text `<input>` form elements that collect information about the user require the autocomplete attribute. The `autocomplete` attribute allows the browser to do a pattern match against a list of values locally stored with the browser, and supplies the appropriate corresponding value when the input is programmatically tagged.

### Proposal

Add all values for the `autocomplete` attribute. A full list can be found at [HTML autocomplete attribute specification](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofilling-form-controls:-the-autocomplete-attribute).

**Comment by justinmc** [comment]
@mdebbar What do we do on the web for autocompletion? This issue seems to imply that we don't use the `autocomplete` attribute on our `<input>` tags at all, is that true?

**Comment by Renzo-Olivares** [comment]
This might be a case where we add a SemanticsProperty that contains an enum for different modes. We cannot use SemanticsFlag because it is strictly boolean.

**Comment by chunhtai** [comment]
This should already supported, if you assign https://api.flutter.dev/flutter/widgets/EditableText/autofillHints.html in editableText or TextField, this should correctly set the autocomplete field **when** the Textfield receive keyboard focus. Text fields that hasn't receive focus before  won't have the autocomplete set. This should be good enough for the feature to work. I am closing bug, feel free to reopen if there is use case we didn't cover
